.github/workflows/release-comments.yml

```diff
@@ -1,11 +1,14 @@
-name: 📝 Comment on Release
+name: Comment on Release
 
 on:
   workflow_call:
     inputs:
       ref:
         required: true
         type: string
+      packageVersionToFollow:
+        required: false
```

Comment:
nightlies publish a single git tag, so we don't need to follow any specific package